src/kirby/components/reorder-list/reorder-list.component.html

```diff
@@ -0,0 +1,30 @@
+<ion-backdrop *ngIf="reorderActive" [tappable]="false" [visible]="true" [stopPropagation]="false">
+</ion-backdrop>
+<ion-reorder-group (ionItemReorder)="doReorder($event)" disabled="false">
+  <div *ngFor="let reorderItem of items">
```

Comment:
The div is there for styling reasons and becasuse of the implementation of ionic reorder.
Ionic uses `transform` for moving the elements and we are also using transform for scaling. If I remove the div ionics transform will win since the use inline styling.

Reply:
I tried adding the trackBy function, but it seems to mess with styling after some elements have been moved around. Could be because I only used index. But in that case you would need to pass in a function because I don't know the type of the items.